### PR TITLE
fix broken doc errors

### PR DESCRIPTION
**Pull request by ckerr**
Fix a handful of errors that prevented `create-typescript-definitions` from running in `master`.

 - [x] Fix missing documentation for methods mentioned in 'breaking-changes.md' that caused the run to error out with "error TS2339"
 - [x] Fix routingId type. Previous phrasing confused the parser. Previously, the string "A unique id" got parsed as `routingId` being of type `unique` :smiley:
 - [x] Tweak Referrer.policy type so that all the possible values were read. Previously, the parser got confused by the word 'or' in 'Can be foo, bar, or mum'.
 - [x] Make LoadURLOptions.Referrer type consistent in the different files. Some definitions had it as a string; others had it as a string | Referrer, which caused the run to fail with `Error: Interface "LoadURLOptions" has already been declared` (cc @jkleinsc)

### CODE REVIEW

File: spec/package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "chai-as-promised": "^7.1.1",
     "coffee-script": "1.12.7",
     "dbus-native": "^0.2.3",
+    "fs-extra": "^6.0.0",
```
Comment (zeke) [inline spec/package.json]: Is this needed?

Comment (ckerr) [reply spec/package.json]: Gah, no that's from an unrelated task that shouldn't've been in this PR. Good catch; will remove

**Comment by MarshallOfSound** [comment]
Will do a follow up PR for this into `electron-typescript-definitions` we don't want to document API's that are deprecated as they will still show up in the defintions.  The best way to force people off the API's is to completely remove them from the docs and therefore from the definitions (it's what we've done in the past)

**Comment by MarshallOfSound** [comment]
Actually there is already a PR that fixes this --> https://github.com/electron/electron-typescript-definitions/pull/101

**Pull request merged**